---
layout: cv
title: Evert Bosdriesz's CV
---
# Evert Bosdriesz

Assistant Professor, Vrije Universiteit Amsterdam

Bioinformatics, Department of Computer Science<br/>

<a href="e.bosdriesz@nki.nl">e.bosdriesz@nki.nl</a> - +31 (0)6 20678153

<div id="webaddress">
  <a href="https://orcid.org/0000-0002-8788-9548"><i class="ai ai-orcid"></i> 0000-0002-8788-9548</a> -
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Bosdriesz+E"><i class="ai ai-pubmed"></i> Pubmed </a> -
  <a href="https://scholar.google.com/citations?user=FWmDPGgAAAAJ&hl=en"><i class="ai ai-google-scholar"></i> Google Scholar</a>
</div>

### Qualities 

Enthusiastic, creative, analytical, collaborative, independent thinker, perseverance.

### Specialized in

Computational cancer biology, statistical analysis of -omics data, (quasi) mechanistic modeling.

### Research question

What determines why and how cancer cells respond to targeted inhibitors?

## Experience

`2014-2019`
**Assistant Professor**, *Bioinformatics, Vrije Universiteit Amsterdam*.<br>

* Organizing MSc course "Bioinformatics for Translational Medicine"
* Track coordinator "Bioinformatics and Systems Biology" MSc program
* Developing network-reconstruction methods.

`2014-2019`
**Postdoctoral Fellow**, *Computational Cancer Biology, Netherlands Cancer Institute*.<br>

* Developed network-reconstruction methods.
* Analysed multi-omics data sets (pre-clinical and clinical).
* Collaborated with many "wet-lab" groups.
* Supervised MSc students and a bioinformatician.
* Member of the Postdoc-committee.
* Gave guest lectures and organized BioSB graduate student course.

`2009-2014`
**PhD Candidate**, *Systems Bioinformatics, Vrije Universiteit Amsterdam*.<br/>

* Analysed ODE-models.
* Identified design principles in regulatory circuits.
* Performed some experiments in yeast.
* Taught various bachelor and master courses.
* Organized NISB seminar series.

`2008-2013` **Various teaching jobs**

* Teaching assistant as MSc student (physics, beta-gamma bachelor)
* Giving exam trainings (physics and math, VMBO and HAVO)
* One-on-one tutoring of high-school students (physics, math and chemistry, HAVO and VWO)

## Education

`2009-2013`
**PhD, Systems Bioinformatics**, *Vrije Universiteit Amsterdam*.<br/>
Thesis: "Darwin`s invisible hand: Optimality principles in cellular resource allocation."<br/>
Advisers: Prof. B. Teusink, Prof. F.J. Bruggeman and Dr. D. Molenaar.

`2006-2009`
**MSc Theoretical Physics (Cum Laude)**, *University of Amsterdam*<br/>
Thesis: "The Central Spin Problem and the Richardson Equations."<br/>
Adviser: Prof. J.S. Caux

`2007-2008` **Erasmus program**, *Humboldt University*, Berlin, Germany.<br/> 

`2003-2006` **BSc Physics and Astrophysics**, *University of Amsterdam*.<br/>
Thesis: "Learning by Reward and Punishment in a Biologically Inspired Neural Network."<br/>
Advisor: Dr. W.A. van Leeuwen

`2002-2003` **Beta-Gamma Propedeuse**, *University of Amsterdam*.<br/>

## Coding

`Proficient`
Python, R, CPLEX, Mathematica

`Intermediate`
bash, snakemake, git

`Rusty`
C, C++ (It's been a while)

## Awards

`2013` **SB@NL symposium poster prize** (2th place)

`2011` **FEBS Youth Travel Fund**, Grant covering expenses to attend the FEBS-SysBio2011 advanced lecture course.

`2010` **Shell Theoretical Physics Stipend**, Awarded annually to the best graduates in theoretical physics in the Netherlands.

`2009` **MSc Theoretical Physics Cum laude**

## Publications

 A list is also available at [google scholar](https://scholar.google.com/citations?user=FWmDPGgAAAAJ&hl=en) or [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/?term=Bosdriesz+E). # indicates shared first author.

### Key

**E. Bosdriesz**, A. Prahallad, B. Klinger, A. Sieber, A. Bosma, R. Bernards, N. Bl&uuml;thgen and L.F.A. Wessels, ‘Comparative Network Reconstruction Using Mixed Integer Programming’, *Bioinformatics*, *34*:i997–1004. doi:[10.1093/bioinformatics/bty616](https://doi.org/10.1093/bioinformatics/bty616), 2018.<br>
*Method to reconstruct and compare signaling networks based on perturbation data using Mixed Integer Quadratic programming.*:
<i class="fab fa-github"></i> <a href="https://github.com/NKI-CCB/cnr"> Code </a> -
<a href="https://github.com/NKI-CCB/CNR-analyses"> Notebooks </a>

A. Ressa#, **E. Bosdriesz**#, J. de Ligt, S. Mainardi, G. Maddalo, A. Prahallad, M. Jager, L. de la Fonteijne, M. Fitzpatrick, S. Groten, A.F.M. Altelaar, R. Bernards, E. Cuppen, L.F.A Wessels and J.R. Heck,
‘A System-Wide Approach to Monitor Responses to Synergistic BRAF and EGFR Inhibition in Colorectal Cancer Cells’,
*Molecular & Cellular Proteomics*, *17*(10):1892-1908.
doi:[10.1074/mcp.RA117.000486](https://doi.org/10.1074/mcp.RA117.000486), 2018.<br>
*Transcriptomic, proteomic and phosphoproteomic, time-course analysis of cancer cells in response to targeted inhibitors*:
<a href="https://bitbucket.org/evertbosdriesz/cgc-multi-omics/src/master/"><i class="fab fa-bitbucket"></i> Code </a>

**E. Bosdriesz**, D. Molenaar, B. Teusink, and F.J. Bruggeman,
‘How Fast-Growing Bacteria Robustly Tune Their Ribosome Concentration to Approximate Growth-Rate Maximization.’,
*FEBS Journal*, *282*(10):2029-2044. doi:[10.1111/febs.13258](https://doi.org/10.1111/febs.13258), 2015.<br>
*Uses mechanistic, minimal models to understand the design principles of how bacteria control their macromolecular composition*:
<a href="https://jjj.bio.vu.nl/models/bosdriesz1/"><i class="fas fa-home"></i> model</a>

### All

`2020`
T. Sustic, **E. Bosdriesz**, S. van Wageningen, L.F.A. Wessels and R. Bernards, 'RUNX2/CBFB modulates the response to MEK inhibitors through activation of receptor tyrosine kinases in *KRAS*-mutant colorectal cancer', *Translational Oncology*, *13*(2):201-211.
doi:[10.1016/j.tranon.2019.10.006(https://doi.org/10.1016/j.tranon.2019.10.006), 2020.


`2018`
**E. Bosdriesz**, A. Prahallad, B. Klinger, A. Sieber, A. Bosma, R. Bernards, N. Blüthgen and L.F.A. Wessels, ‘Comparative Network Reconstruction Using Mixed Integer Programming’, *Bioinformatics*, *34*:i997–1004. doi:[10.1093/bioinformatics/bty616](https://doi.org/10.1093/bioinformatics/bty616), 2018.

A. Ressa#, **E. Bosdriesz**#, J. de Ligt, S. Mainardi, G. Maddalo, A. Prahallad, M. Jager, L. de la Fonteijne, M. Fitzpatrick, S. Groten, A.F.M. Altelaar, R. Bernards, E. Cuppen, L.F.A Wessels and J.R. Heck ,
‘A System-Wide Approach to Monitor Responses to Synergistic BRAF and EGFR Inhibition in Colorectal Cancer Cells’,
*Molecular & Cellular Proteomics*, *17*(10):1892-1908.
doi:[10.1074/mcp.RA117.000486](https://doi.org/10.1074/mcp.RA117.000486), 2018.

T. Šuštić#, S. van Wageningen#, **E. Bosdriesz**, R.J.D. Reid, J. Dittmar, C. Lieftink,  R.L. Beijersbergen, L.F.A. Wessels, R. Rothstein and R. Bernards, ‘A Role for the Unfolded Protein Response Stress Sensor ERN1 in Regulating the Response to MEK Inhibitors in KRAS Mutant Colon Cancers’,
*Genome Medicine*, *10*:90. doi:[10.1186/s13073-018-0600-z](https://doi.org/10.1186/s13073-018-0600-z), 2018.

L. Wang#, R. Leite de Oliveira#, S. Huijberts, **E. Bosdriesz**, N. Pencheva, D. Brunen, A. Bosma, J.Y. Song, J. Zevenhoven, G. T. Los-de Vries, H. Horlings, B. Nuijen, J.H. Beijnen, J.H.M. Schellens and R. Bernards.
‘An Acquired Vulnerability of Drug-Resistant Melanoma with Therapeutic Potential’.
*Cell*, *173*(6):1413-1425.e14. doi:[10.1016/j.cell.2018.04.012](https://doi.org/10.1016/j.cell.2018.04.012), 2018.

M. Dorel, B. Klinger, T. Gross, A. Sieber, A. Prahallad, **E. Bosdriesz**, L.F.A. Wessels and N. Blüthgen, ‘Modelling Signalling Networks from Perturbation Data’, *Bioinformatics*, *34*(23):4079–4086. doi:[10.1093/bioinformatics/bty473](https://doi.org/10.1093/bioinformatics/bty473), 2018.

Z. Xue, D.J. Vis, A. Bruna, T. Šuštić, S. Van Wageningen, A. Sati Batra, O.M. Rueda, **E. Bosdriesz**, C. Caldas, L.F.A. Wessels and R. Bernards. ‘MAP3K1 and MAP2K4 Mutations Are Associated with Sensitivity to MEK Inhibitors in Multiple Cancer Models’.
*Cell Research*, *28*:719–729. doi:[10.1038/s41422-018-0044-4](https://doi.org/10.1038/s41422-018-0044-4), 2018.

**E. Bosdriesz**#, M.T. Wortel#, J.R. Haanstra, M.J. Wagner, P. de la Torre Cortés, and B. Teusink, ‘Low Affinity Uniporter Carrier Proteins Can Increase Net Substrate Uptake Rate by Reducing Efflux’, *Scientific Reports*, *8*:5576. doi:[10.1038/s41598-018-23528-7](https://doi.org/10.1038/s41598-018-23528-7), 2018.

`2016`
M.T. Wortel#, **E. Bosdriesz#**, B. Teusink, and F.J. Bruggeman.
‘Evolutionary Pressures on Microbial Metabolic Strategies in the Chemostat’.
*Scientific Reports*, *6*:29503. doi:[10.1038/srep29503](https://doi.org/10.1038/srep29503), 2016.

`2015`
**E. Bosdriesz**, D. Molenaar, B. Teusink, and F.J. Bruggeman, ‘How Fast-Growing Bacteria Robustly Tune Their Ribosome Concentration to Approximate Growth-Rate Maximization.’, *FEBS Journal*, *282*(10):2029-2044. doi:[10.1111/febs.13258](https://doi.org/10.1111/febs.13258), 2015.

**E. Bosdriesz**, S. Magnúsdóttir, F.J. Bruggeman, B. Teusink, and D. Molenaar,
‘Binding Proteins Enhance Specific Uptake Rate by Increasing the Substrate-Transporter Encounter-Rate’,
*FEBS Journal*, *282*:2394–2407. doi:[10.1111/febs.13289](https://doi.org/10.1111/febs.13289), 2015.

`2013`
J. Berkhout, **E. Bosdriesz**, E. Nikerel, D. Molenaar, D. de Ridder, B. Teusink and F.J. Bruggeman,
‘How Biochemical Constraints of Cellular Growth Shape Evolutionary Adaptations in Metabolism.’,
*Genetics*, *194*:505–512. doi:[10.1534/genetics.113.150631](https://doi.org/10.1534/genetics.113.150631), 2013.

<br>

### Preprints

J.M. Fernandes Neto,E. Nadal, S.N. Ooft, **E. Bosdriesz**, L. Farre,C. McLean, S. Klarenbeek, A. Jurgens, H. Hagen, L. Wang, E. Felip, A. Martinez-Marti, E. Voest, L.F.A. Wessels, O. van Tellingen, A. Villanueva and R. Bernards.
'Multiple Low Dose (MLD) therapy: an effective strategy to treat EGFR inhibitor-resistant NSCLC tumours', *BioRxiv* 821975. doi:[10.1101/821975](https://.doi.org/10.1101/821975), 2019.

N. Aben, J. de Ruiter, **E. Bosdriesz**, Y. Kim, G. Bounova, D.J. Vis, L.F.A. Wessels and M. Michaut.
‘Identifying Biomarkers of Anti-Cancer Drug Synergy Using Multi-Task Learning.’
*BioRxiv* 243568. doi:[10.1101/243568](https://doi.org/10.1101/243568), 2018.<br>


## Talks

`2018`
**INCOME2018**,
*Comparative network reconstruction to identify selective anti-cancer drug combinations*,
Bernried, Germany.

**17th European Conference on Computational Biology**,
*Comparative Network Reconstruction using Mixed Integer Progamming*,
Athens, Greece.

**VUmc seminar series on bioinformatics and data analysis**, *Comparative network reconstruction to identify selective anti-cancer drug combinations.*, Amsterdam, the Netherlands.

**BioSB 2018**,
*Comparative network reconstruction to identify selective anti-cancer drug combinations*,
Lunteren, the Netherlands.

`2016`
**BioSB 2016**,
*Comparative network reconstruction identifies resistance mechanisms to targeted cancer treatment*,
Lunteren, the Netherlands.

`2014`
**EMBO: From Functional Genomics to Systems Biology**,
*The logic of the regulatory mechanism underlying growth rate control in Escherichia coli*,
Heidelberg, Germany.

`2013`
**14th International Conference on Systems Biology**,
*Escherichia coli implements a robust regulatory network motif that maximizes growth rate*,
Copenhagen, Denmark.

**Netherlands Bioinformatics Conference 2013**,
*Design principles of nutrient-uptake systems involving binding proteins*,
Lunteren, the Netherlands.

`2012`
**Netherlands Consortium for Systems Biology symposium**,
*Escherichia coli implements a robust regulatory network motif that maximizes growth rate*,
Soesterberg, the Netherlands.

**Netherlands Bioinformatics Conference 2012**,
*Optimal and robust regulation of gene expression*,
Lunteren, the Netherlands.

**Netherlands Biotechnology Congress**,
*Optimal and robust regulation of gene expression*,
Ede, the Netherlands.

## Reviewing

`Independent`
Bioinformatics, Cellular Oncology, Cancer Medicine, Molecular Cancer Therapeutics, Scientific Reports.

`Co-reviewed`
Nature Communications, Plos Computational Biology (with Prof. Wessels),
Lung Cancer (with Prof. Bernards),
Biophysical Journal (with Prof. Bruggeman).
